v3/src/components/mosaic-tile-row.tsx

```diff
@@ -17,7 +17,8 @@ interface IMosaicTileRowProps {
   content?: IDocumentContentModel
   getTile: (tileId: string) => ITileModel | undefined
 }
-export const MosaicTileRowComponent = observer(({ content, row, getTile }: IMosaicTileRowProps) => {
+export const MosaicTileRowComponent = observer(function MosaicTileRowComponent(
+  { content, row, getTile }: IMosaicTileRowProps) {
```

Comment:
```suggestion
export const MosaicTileRowComponent = observer(
  function MosaicTileRowComponent({ content, row, getTile }: IMosaicTileRowProps) {
```
Here and elsewhere in this file, as a matter of personal preference, I think I'd be inclined to keep the function declaration together in this situation, but others may fee differently.